core/chains/evm/logpoller/log_poller.go

```diff
@@ -347,6 +351,18 @@ func (lp *logPoller) run() {
 				start = lastProcessed.BlockNumber + 1
 			}
 			lp.pollAndSaveLogs(lp.ctx, start)
+		//case <-logPollFinalizedTick:
+		//	if lp.nextFinalizedBlockNum == 0 {
+		//		lp.lggr.Warnw("backup log poller triggered before any blocks were fetched")
+		//		continue
+		//	}
+		//	lp.lggr.Debugw("Polling for finalized logs", "nextFinalizedBlockNum", lp.nextFinalizedBlockNum)
+		//	latestBlock, err := lp.ec.HeadByNumber(lp.ctx, nil)
+		//	if err != nil {
+		//		lp.lggr.Warnw("Unable to get latestBlockNumber block", "err", err, "nextFinalizedBlockNum", lp.nextFinalizedBlockNum)
+		//		return
+		//	}
+		//	lp.backfillFinalizedLogs(lp.ctx, latestBlock.Number)
```

Comment:
?

Reply:
Oops, remnant of the original design... I'll remove it.